tegola/tegola_st_asbinary.toml

```diff
@@ -0,0 +1,362 @@
+[[providers]]
```

Comment:
I believe this is not going to be ever used. The reason for the ST_AsBinary function is to provide the proper geometry for mapnik to generate the vector-tile. It even breaks integration with the raster endpoint.

Reply:
Ok. I can remove it. I was keeping it in here for reference, but may as well make this repo focus on the current trajectory.

Reply:
Fixed!